src/terrain/clients/iplant_groups.clj

```diff
@@ -49,21 +49,14 @@
     user-info
     (empty-user-info short-username)))
 
-(defn search-subjects
-  "Uses iplant-groups's subject search endpoint to retrieve user details."
-  [user search]
-  (let [res (http/get (str (curl/url (config/ipg-base) "subjects"))
-                      ;; Adding wildcards matches previous (trellis) search behavior
-                      {:query-params {:user user :search (str "*" search "*")}
```

Comment:
Yeah, we've moved to a slightly newer version of Grouper since then, but that's the only change that I can think of that would have caused this.